docs/Developer Resources/illumos Exceptions.rst

```diff
@@ -1,31 +1,30 @@
-OpenZFS Exceptions
+illumos Exceptions
 ==================
 
 Commit exceptions used to explicitly reference a given Linux commit.
 These exceptions are useful for a variety of reasons.
 
 **This page is used to generate**
-`OpenZFS Tracking <http://build.zfsonlinux.org/openzfs-tracking.html>`__
+`illumos Tracking <http://build.zfsonlinux.org/illumos-tracking.html>`__
```

Comment:
It's regeneration is disabled now and it remains mainly for history. Before URL change we should rename file itself, which we cannot do via this repo only. Let's remain it for now.